homeassistant/components/eq3btsmart/config_flow.py

```diff
@@ -47,46 +51,47 @@ async def async_step_user(
         self._abort_if_unique_id_configured(updates=user_input)
 
         # We can not validate if this mac actually is an eQ-3 thermostat,
-        # since the thermostat probably is not advertising right now.
-        return self.async_create_entry(title=slugify(mac_address), data={})
+        # since the thermostat might not be advertising right now.
+        return self.async_create_entry(
+            title=slugify(mac_address), data={CONF_MAC_ADDRESS: mac_address}
+        )
```

Comment:
re: the mac address in the data change, I think we can do that today already in a separate PR. I don't see yet why the coordinator change would require this